packages/components/upload/__tests__/upload.test.tsx

```diff
@@ -161,6 +169,7 @@ describe('<upload />', () => {
 
       expect(beforeUpload).toHaveBeenCalled()
       await flushPromises()
+      expect(httpRequest).toHaveBeenCalled()
```

Comment:
i notice this error in my local env, but don't know how to adjust the test code. `httpRequest` is not called!
![image](https://user-images.githubusercontent.com/24362871/235098147-9cf977ab-0872-4411-9857-389624f7a01c.png)